src/app/shell/details/competition-details/competition-details.component.spec.ts

```
@@ -0,0 +1,108 @@
+      schemas: [NO_ERRORS_SCHEMA]
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CompetitionDetailsComponent);
+    component = fixture.componentInstance;
+    component.competition = {} as Competition;
+    component.provider = {} as Provider;
+    matDialog = TestBed.inject(MatDialog);
+    fixture.detectChanges();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)